doc/api/magic-plot.md

```diff
@@ -0,0 +1,154 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# `%sqlplot`
+
+```{versionadded} 0.5.2
+```
+
+
+```{note}
+`%sqlplot` requires `matplotlib`: `pip install matplotlib` and this example requires
```

Comment:
Don't we want here ske instead of mplotlib?

Reply:
why would we use ske? we're not plotting ML models, we're using it for boxplots and histograms

Reply:
Yeah fair point, I saw the plots I thought it might be useful here.